### PR TITLE
Add simple instructions on how to use Bigtable Change Stream API

### PR DESCRIPTION
**Pull request by tonytanger**
Add simple instructions on how to use Bigtable Change Stream API

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @Abacn 

Are you a good reviewer?

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java [hunk 1]
```diff
@@ -182,6 +182,50 @@
...
+ * <p>Change stream connector creates and manages a metadata table to manage the state of the
+ * connector. By default, the table is created in the same instance as the table being streamed.
+ * However, it can be overridden with {@link
+ * BigtableIO.ReadChangeStream#withMetadataTableProjectId}, {@link
+ * BigtableIO.ReadChangeStream#withMetadataTableInstanceId}, {@link
+ * BigtableIO.ReadChangeStream#withMetadataTableTableId}, and {@link
+ * BigtableIO.ReadChangeStream#withMetadataTableAppProfileId}. The app profile for the metadata
+ * table must be a single cluster app profile with single row transaction enabled.
+ *
+ * <p>Note - To prevent unforeseen stream stalls, the Dataflow connector outputs all data with an
```
Comment (Abacn) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java:201]: what is "Dataflow connector"? do you mean BigtableIO connector?

**Review by Abacn — CHANGES_REQUESTED** [review]
Thanks! Just found 2 minor points.

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java [hunk 2]
```diff
@@ -182,6 +182,50 @@
  *
  * }</pre>
  *
+ * <h3>Streaming Changes from Cloud Bigtable</h3>
+ *
+ * <p>Cloud Bigtable change streams enable users to capture and stream out mutations from their
+ * Cloud Bigtable tables in real-time. Cloud Bigtable change streams enable many use cases including
+ * integrating with a user’s data analytics pipelines, support audit and archival requirements as
```
[the remark this answers is not in the document]
Comment (tonytanger) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java]: Thanks. Done!

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java (hunk 1 above)
Comment (tonytanger) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java]: Yes. Thanks for catching this.

**Review by Abacn — APPROVED** [review]
Thanks, this lgtm

**Pull request merged**